src/tye/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
nit: revert this file change.

Reply:
Do you mean the file: launchSettings.json should be deleted?

I made a copy of the single-phase-dockerfile test folder, which also contains a file like this and the contents are the same.  I can change the ports or other details in the file if you think the two tests could end up clashing

Reply:
Removed